src/exportable/ExportableValueSetComponentRule.ts

```diff
@@ -15,6 +16,10 @@ export class ExportableValueSetConceptComponentRule extends fshrules.ValueSetCon
     let fsh = `* ${this.inclusion ? 'include' : 'exclude'} `;
     fsh += this.concepts.map(concept => concept.toString()).join(' and ');
     fsh += fromString(this.from);
+    // if the result is more than 100 characters long, break it up
+    if (fsh.length > 100) {
+      fsh = fsh.replace(/ (and|from) /g, `${EOL}  $1 `);
```

Comment:
I think this may be an overly aggressive replacement. Codes may have displays, and those displays could have the words `and` and `from` in them, resulting in output that looks like this:
![image](https://user-images.githubusercontent.com/52747882/99972948-30e60c00-2d6d-11eb-8be7-b70f46d31ed5.png)

Which doesn't cause errors in SUSHI, but does change the `display` produced by SUSHI.

Once [CIMPL-599](https://standardhealthrecord.atlassian.net/browse/CIMPL-599) is done, we could run into codes that have spaces in them, and those codes could have `and`, `from`, or `where` in them. In that case, if GoFSH splits the quoted code across multiple lines, it will cause errors in SUSHI.